### PR TITLE
Add info about using env vars in Agent reference

### LINKED ISSUE #1992: Agent reference should better describe possible uses for environment variables
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior

Documentation describes potential uses for Sensu Agent, e.g. service configuration, setting environment variables which are available to plugins.

## Current Behavior

[Agent reference describes configuration via environment variables](https://docs.sensu.io/sensu-go/5.15/reference/agent/#configuration-via-environment-variables) but does not explain that these environment variables are also passed to plugin processes. 

## Possible Solution

Expand this section of the agent reference.

## Affected Docs Pages

[Agent reference](https://docs.sensu.io/sensu-go/5.15/reference/agent/) (all versions, I think)

## Context

image

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Added section about using environment variables provided in the sensu-agent file in the Agent reference.
Content summarized from discussion at https://sensucommunity.slack.com/archives/C68LV5M9U/p1575497795036400?thread_ts=1575491274.034100&cid=C68LV5M9U

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/1992

## Review Instructions
The section I added still seems thin -- is there anything else to add?

### CODE REVIEW

File: content/sensu-go/5.16/reference/agent.md [hunk 1]
```diff
@@ -1305,6 +1306,14 @@ $ sudo systemctl restart sensu-agent
 
 {{< /language-toggle >}}
 
+## Use environment variables with the Sensu agent
+
+After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to your Sensu resources.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to commands executed by Sensu resources, including handlers and checks.
```

File: content/sensu-go/5.16/reference/agent.md [hunk 2]
```diff
@@ -1305,6 +1306,14 @@ $ sudo systemctl restart sensu-agent
...
+This includes your checks and plugins.
+
+For example, if you configure a `SENSU_TEST_VAR` variable in your sensu-agent file, it will be available to use in your check configurations as `$SENSU_TEST_VAR`.
+
+The environment variables you provide in your sensu-agent file are also available for plugin processes.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

File: content/sensu-go/5.16/reference/agent.md (hunk 2 above)
Comment (rgeniesse) [inline content/sensu-go/5.16/reference/agent.md]: This line seems redundant as on line 1312 it is mentioned that this works for plugins as well. Line 1314 is needed as it shows a more concrete example how to use them.

Comment (rgeniesse) [reply content/sensu-go/5.16/reference/agent.md]: Resolving as it has been removed.

**Review by rgeniesse — APPROVED** [review]
LGTM now.

File: content/sensu-go/5.16/reference/agent.md [hunk 3]
```diff
@@ -1305,6 +1306,13 @@ $ sudo systemctl restart sensu-agent
 
 {{< /language-toggle >}}
 
+## Use environment variables with the Sensu agent
+
+After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to commands executed by Sensu resources, including handlers and checks.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/reference/agent.md]:
```suggestion
After you [configure][50] your sensu-agent service to read environment variables from `/etc/default/sensu-agent` (Debian/Ubuntu) or `/etc/sysconfig/sensu-agent` (RHEL), any environment variables you provide there will be available to check and hook commands executed by the Sensu agent.
```

I just remembered agents don't run handlers, so they arent a good example here.

**Pull request merged**